public-api/src/main/java/org/pmiops/workbench/cdr/dao/QuestionConceptDao.java

```diff
@@ -7,9 +7,12 @@
 
 public interface QuestionConceptDao extends CrudRepository<QuestionConcept, Long> {
 
-    @Query(nativeQuery=true, value="SELECT c.concept_id, c.concept_name, c.domain_id, c.vocabulary_id, c.concept_code, " +
-            "c.count_value, c.prevalence  " +
-            "from concept c join concept_relationship  r on c.concept_id = r.concept_id_2 " +
-            "where r.concept_id_1=?1 and r.relationship_id = 'Module of' ")
+    @Query(nativeQuery=true, value="SELECT C.concept_id,C.concept_name,C.domain_id,C.vocabulary_id,C.concept_code,C.count_value,C.prevalence from\n" +
+            "(SELECT c.*,count(*) as result_count from\n" +
+            "concept c join concept_relationship r on c.concept_id = r.concept_id_2\n" +
+            "join achilles_results ar on ar.stratum_2=c.concept_id\n" +
+            "where r.concept_id_1=?1 and r.relationship_id = 'Module of' and ar.stratum_1=?1\n" +
+            "group by c.concept_id,c.concept_name,c.domain_id,c.vocabulary_id,c.concept_code,c.count_value,c.prevalence) as C\n" +
+            "where result_count > 0")
```

Comment:
I'm not a fan of this select from ( select ... ) ... It is slower too. I tried this: and it is much faster. 

SELECT c.concept_id,c.concept_name,c.domain_id,c.vocabulary_id,c.concept_code,c.count_value,c.prevalence from concept c join concept_relationship  r on c.concept_id = r.concept_id_2
join achilles_results ar on ar.stratum_2=c.concept_id
where r.concept_id_1=1586134 and r.relationship_id = 'Module of' and ar.analysis_id = 3110
group by c.concept_id,c.concept_name,c.domain_id,c.vocabulary_id,c.concept_code,c.count_value,c.prevalence

Just do you join and group by and i think it is simpler to read too.

Reply:
Above is 89 ms.  This one you have below is 210ms. 
SELECT C.concept_id,C.concept_name,C.domain_id,C.vocabulary_id,C.concept_code,C.count_value,C.prevalence from
    (SELECT c.*,count(*) as result_count from
     concept c join concept_relationship r on c.concept_id = r.concept_id_2
     join achilles_results ar on ar.stratum_2=c.concept_id
     where r.concept_id_1=1586134 and r.relationship_id = 'Module of' and ar.stratum_1="1586134" 
     group by c.concept_id,c.concept_name,c.domain_id,c.vocabulary_id,c.concept_code,c.count_value,c.prevalence) as C
  where result_count > 0;
210ms ...